packages/rspack-browser/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
maybe we should add some description of @rspack/browser and claims it doesn't follow semver now

Reply:
We should also add a MIT license file to this folder